packages/calcite-components/src/components/accordion-item/accordion-item.tsx

```diff
@@ -349,6 +351,10 @@ export class AccordionItem extends LitElement {
             tabIndex="0"
           >
             <div class={CSS.headerContainer}>
+              <div class={CSS.slotContentStart}>
```

Comment:
Should this container be hidden until something is slotted inside it? I notice it has as margin-inline-end.